### PR TITLE
feat(ingest): print detailed GMS error messages

### PR DESCRIPTION
**Pull request by djordje-mijatovic**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/cli/cli_utils.py [hunk 1]
```diff
@@ -581,6 +581,9 @@ def post_entity(
         curl_command,
     )
     response = session.post(url, payload)
+    jsonResponse = json.loads(response.content)
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/cli/cli_utils.py:584]: Can we check whether response.content exists?

Also can we use response.json() instead?

Comment (djordje-mijatovic) [reply metadata-ingestion/src/datahub/cli/cli_utils.py]: Yea, we can use response.json(). In that case there is no need to check response.content as we do not use response.content at all.

File: metadata-ingestion/src/datahub/cli/cli_utils.py [hunk 2]
```diff
@@ -581,6 +581,9 @@ def post_entity(
         curl_command,
     )
     response = session.post(url, payload)
+    response_json = response.json()
+    if 'message' in response_json:
```
Comment (szalai1) [inline metadata-ingestion/src/datahub/cli/cli_utils.py:585]: you can use `if not response.ok` instead, so you don't have to rely on the structure of the message.

Comment (jjoyce0510) [reply metadata-ingestion/src/datahub/cli/cli_utils.py:585]: This may be safer^

**Review by hsheth2 — APPROVED** [review]
Fixed up the error handling a bit but LGTM

**Review by szalai1 — APPROVED** [review]

**Pull request merged**